backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/model/SubscriptionCreateModel.java

```diff
@@ -18,6 +19,7 @@ public interface SubscriptionCreateModel {
 	String getAlertTypeCode();
 
 	@Schema(example = "1033")
-	Long getPreferredLanguage();
+	@LanguageCode(message = "Preferred language code does not exist")
+	String getPreferredLanguageCode();
```

Comment:
I wonder if this should be `getMsLanguageCode()` instead, since there are different types of codes for each language?

Reply:
updated